app/scripts/modules/core/src/presentation/formsy/components/Input.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+
+import { FormComponent, IFormComponentState, IFormComponentProps } from '../FormComponent';
+
+export interface IProps extends IFormComponentProps {
```

Comment:
Where did we land on exported props names? I like the pattern of if its exported to name it...